src/System.Runtime.Serialization.Formatters/src/System.Runtime.Serialization.Formatters.csproj

```diff
@@ -60,19 +60,14 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
-    <Reference Include="System.Collections" />
-    <Reference Include="System.Collections.Concurrent" />
-    <Reference Include="System.Collections.NonGeneric" />
-    <Reference Include="System.Diagnostics.Debug" />
-    <Reference Include="System.Memory" />
-    <Reference Include="System.Resources.ResourceManager" />
-    <Reference Include="System.Runtime" />
-    <Reference Include="System.Runtime.CompilerServices.Unsafe" />
-    <Reference Include="System.Runtime.Extensions" />
-    <Reference Include="System.Text.Encoding.Extensions" />
-    <Reference Include="System.Threading" />
+    <ReferenceFromRuntime Include="System.Private.CoreLib" />
+    <ProjectReference Include="..\..\System.Runtime.CompilerServices.Unsafe\src\System.Runtime.CompilerServices.Unsafe.ilproj" />
```

Comment:
Why changing this to p2p references? cc @ericstj

Reply:
Good point: if this uses ReferenceFromRuntime that means that it depends on System.Private.CoreLib internals, that means we should no longer ship it in a package and it has to be part of the shared framework.  I don't think we want to do that for BinaryFormatter.

Reply:
BinaryFormatter is already part of the shared framework or are you talking about CompilerServices.Unsafe?